README.md

```diff
@@ -60,32 +63,31 @@ Steps to get started:
     - Use the [web console](https://console.ng.bluemix.net/dashboard/cf-apps) and download the starter code
     - Create a new CloudFoundry app manually
 - Open the *project’s folder* locally in the VS Code editor
-- Use `bx cf apps` to list all of your apps
-- Use `bx cf push` to push a build of your app
-- Use `bx cf <start/stop/restage/restart>` to change the status of your app
-- Use `bx cf logs` to view the live log stream for your app
-  - _Use `bx cf logs` to stop the log stream_
+- Use `ibmcloud cf apps` to list all of your apps
+- Use `ibmcloud cf push` to push a build of your app
+- Use `ibmcloud cf <start/stop/restage/restart>` to change the status of your app
+- Use `ibmcloud cf logs` to view the live log stream for your app
+  - _Use `ibmcloud cf logs` to stop the log stream_
 
 ### Supported CLI/plugins:
 
-- `bx login/logout` - Bluemix user authentication
-- `bx dev` - Bluemix developer CLI
-- `bx cf` - Bluemix CloudFoundry CLI
-- `bx cs` - Bluemix Container Service CLI
-- `bx sdk` - Bluemix SDK Generation CLI
+- `ibmcloud login/logout` - IBMCloud user authentication
+- `ibmcloud dev` - IBMCloud developer CLI
+- `ibmcloud cf` - IBMCloud CloudFoundry CLI
+- `ibmcloud cs` - IBMCloud Container Service CLI
```

Comment:
This is now `ibmcloud ks` - IBM Cloud Kubernetes Service CLI

Reply:
Will do

Reply:
Please change to `IBM Cloud Kubernetes Service CLI`

Reply:
Sorry I must have missed this here. Will fix.